modular-docs-manual/content/topics/module_creating-assemblies.adoc

```diff
@@ -0,0 +1,32 @@
+[[creating-assemblies]]
+= Creating Assemblies
+
+An assembly is a representation of a user story, so you need to create an assembly for each user story that you defined.
+
+An assembly could represent an article, "chapter" in a book, or even an entire book. However, one of the benefits of modular documentation is that you do not need to worry about how the assembly will ultimately be used -- each assembly represents a user goal, and once you create it, it can be "included" anywhere it is needed (a publication, within another assembly, and so on).
+
+.Procedure
+
+. Create an assembly file for each user story that you identified.
++
+--
+Be sure to follow a consistent file naming standard. For example, you could use the following file naming pattern:
+
+`assembly_configuring-security.adoc`
+--
+
+. For each top-level assembly file that you created, fill in the content:
++
+--
+* Title
+* Intro paragraphs
+* Prerequisites
+* Modules or Assemblies
++
+If the top-level assembly has second-level assemblies, then add the names of the second-level assemblies. Otherwise, list the modules.
```

Comment:
"then add the names of the second-level assemblies"  Hm, when I want to include an assembly, I do it just like for modules. This is what my assembly files contain in this case:

~~~~
include::procedure-creating-a-replica.adoc[leveloffset=+1]
include::procedure-stopping-an-ipa-server-instance.adoc[leveloffset=+1]
include::assembly-uninstalling-an-ipa-server.adoc[leveloffset=+1]
~~~~

I don't put the names of the assemblies or modules (their headings) into the assembly file that includes -- I want the heading to be part of the assembly or module file itself so that it can be reused along with the assembly or module itself. That's also what our templates suggest. Or am I misunderstanding something?

Reply:
No, you had it right - I was trying to express what you said here, I just didn't word it very clearly. My intent with the mod docs conversion workflow was to be very generic (and not get into any technology, markup, or syntax-related commentary). If we want this to be asciidoc-specific, we would need to add a step for the master.adoc file. You would include all of the top-level assemblies there. Then you would open each top-level assembly one at a time and include the necessary modules and/or supporting assemblies. At this point, though, I didn't want to get into that level of detail (though we certainly could).